tests/test_ir.py

```diff
@@ -851,10 +851,9 @@ def test_op_custom_verify_is_done_last():
     a = Constant.from_int_and_width(1, i32)
     # CustomVerify expects a i64, not i32
     b = CustomVerify.get(a.result)
-    with pytest.raises(Exception) as e:
+    with pytest.raises(VerifyException) as e:
```

Comment:
Since we are refactoring this a bit:
```suggestion
    with pytest.raises(VerifyException, matches="Custom Verification Check") as e:
```

Reply:
The following assert checks that this is ***not*** present in the error messages (Testing that another VerifyException is raised before that one)
I'm not sure how to best refactor this with pytest TBH